### ISSUE TITLE
SlevomatCodingStandard.TypeHints.ParameterTypeHint incorrectly marks MissingTraversableTypeHintSpecification error

### ISSUE THREAD

**Issue by martin-heralecky** [slevomat/coding-standard#1146]
When the `SlevomatCodingStandard.TypeHints.ParameterTypeHint` sniff is applied to this piece of code, it produces a `MissingTraversableTypeHintSpecification` error saying _"Function a() does not have @param annotation for its traversable parameter $b."_.

```php
/**
 * @param int   $a )
 * @param int[] $b
 */
function a(int $a, array $b)
{
}

```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/8b4ced3066743e3e6dc415660ff04c5e6c8df877
